src/Symfony/Component/HttpKernel/Tests/Controller/ArgumentResolverTest.php

```diff
@@ -331,6 +385,22 @@ protected function controllerWithSessionInterface(SessionInterface $session)
     protected function controllerWithExtendingSession(ExtendingSession $session)
     {
     }
+
+    protected function controllerWithIntTypeHint(int $int)
```

Comment:
this file should be configured with `declare(strict_types=1);` in order to verify both behaviors.

Reply:
can't you create a fixture with the code to actually test run in strict mode?

Reply:
yes, but the caller (the test class) still has to enable strict_types.

Reply:
can you make this test class strict? (with a comment, since that's not the default policy)

Reply:
yes I will do that (it seems to have no influence on other tests) so I guess it's OK.

Reply:
ah, sorry for losing your time, actually we don't need strict_types=1 in the test class because we never invoke the controllers! we just use their reflection, so there is no need to bother.

Reply:
We still might have to add an integration test to cover all the possible behaviors by actually invoking the controllers and all ?